### PR TITLE
Links

### PR DESCRIPTION
**Pull request by jukent**
This PR comments out the Notebook gallery since those aren't real resources we want to display to the public and adds a links.md page containing information from the Content Catalog Google Sheet.

### CODE REVIEW

**Comment by jukent** [comment]
@kmpaul @clyne @brian-rose This PR is ready for review. It looks as though I do not have permission to request reviewers.

**Comment by brian-rose** [comment]
Looks like the home page is corrupted:
https://projectpythia--20.org.readthedocs.build/en/20/index.html

**Comment by jukent** [comment]
> Looks like the home page is corrupted:
> https://projectpythia--20.org.readthedocs.build/en/20/index.html

Good catch! I was so focused on checking the appearance of links.html.

I'll fix this now!

**Review by clyne — APPROVED** [review]
Thanks for the quick turnaround on this, @jukent 

@kmpaul @brian-rose just a quick reminder that I'm hoping to have this PR merged in time for AMS this coming week if possible.

File: content/pages/links.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Link to External Resources
+
+The members of Project Pythia approve of these resources on Python specific topics.
```
Comment (dopplershift) [inline content/pages/links.md:3]: Approve seems a bit strong. How about "Project Pythia find the links below to be useful resources on a variety of Python topics." ?

**Review by dopplershift — APPROVED** [review]
One comment, but I wouldn't hold this up for it.

File: content/pages/links.md (hunk 1 above)
Comment (brian-rose) [reply content/pages/links.md:3]: I think that would be better, avoiding appearance of some ill-defined "approval" process.

But otherwise all looks good.

**Review by brian-rose — APPROVED** [review]

File: content/pages/links.md (hunk 1 above)
Comment (jukent) [reply content/pages/links.md:3]: Good idea! I was uncertain on the best language to use.

**Review by brian-rose — APPROVED** [review]
Great, I think it's ready to merge.

File: content/pages/links.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Link to External Resources
```
Comment (kmpaul) [inline content/pages/links.md:1]: Should be plural...

```suggestion
# Links to External Resources
```

File: content/pages/links.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Project Pythia find the links below to be useful resources on a variety of Python specific topics.
```
Comment (kmpaul) [inline content/pages/links.md:3]:
```suggestion
Project Pythia finds the links below to be useful resources on a variety of Python specific topics.
```

**Review by kmpaul — CHANGES_REQUESTED** [review]
Just a few grammar suggestions...

**Review by erogluorhan — APPROVED** [review]
Looks good to me

**Review by kmpaul — APPROVED** [review]
Great!  Thanks!

**Comment by clyne** [comment]
Thanks for the quick turnaround, All!

**Pull request merged**